### PR TITLE
[MB-3767] Populate home screen data

### PR DESCRIPTION
**Pull request by Ronolibert**
## Description

Populates home screen data when different pieces of data are available

## Reviewer Notes
- Did not support PPMs, I assumed that we'd be doing that elsewhere so I just loaded the HHG Shipments
- Shipment list is currently displaying the ID until we have a better identifier to put in the middle of the list item

## Setup
`make server_run`
`make client_run`
Go through setup process
Visit `/home-2` in a separate tab
When you complete each step (Profile complete, finish uploading orders, finish setting up an HHG), refresh /home-2 tab to see if data is populated correctly

## Code Review Verification Steps
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-3767) for this change

### CODE REVIEW

File: src/pages/MyMove/Home/index.jsx [hunk 1]
```diff
@@ -11,109 +12,198 @@ import Step from 'components/Customer/Home/Step';
...
   componentDidMount() {
-    const { showLoggedInUser } = this.props;
+    const { showLoggedInUser, loadMTOShipments, move } = this.props;
     showLoggedInUser();
+    loadMTOShipments(move.id);
+  }
+
+  get hasOrders() {
+    const { orders, uploadedOrderDocuments } = this.props;
+    return !!Object.keys(orders).length && !!uploadedOrderDocuments.length;
```
Comment (kimallen) [inline src/pages/MyMove/Home/index.jsx:39]: I like how you did this better than how I did it- much cleaner

File: src/pages/MyMove/Home/index.jsx [hunk 2]
```diff
@@ -11,109 +12,198 @@ import Step from 'components/Customer/Home/Step';
...
+          actionBtnDisabled={!this.hasShipments}
           actionBtnLabel="Review and submit"
           containerClassName="margin-bottom-8"
           headerText="Confirm move request"
           onActionBtnClick={() => console.log('some action')}
           step="4"
         >
-          <p className={styles.description}>
+          <Description>
             Review your move details and sign the legal paperwork, then send the info on to your move counselor
```
Comment (kimallen) [inline src/pages/MyMove/Home/index.jsx:205]: We will need the date the move was submitted here after signing.  Is that available on the currently loaded data?  (I don't see it on the move model, wondering if we need to add a field or if it's somewhere else)

**Comment by sojeri** [comment]
@Ronolibert this doesn't support zero state for me where even step 1 should be a number. instead, it crashes on home2 if I browse there right after login

Screen Shot 2020-08-31 at 11 56 46 AM

**Comment by sojeri** [comment]
@Ronolibert if I finish entering the origin duty station and refresh, then I crash here:

Screen Shot 2020-08-31 at 11 59 19 AM

**Comment by sojeri** [comment]
note that before the crashes, I can see the correct state for a moment. :) once orders are in the crashes stop. but before then I see the two crashes posted above.

**Comment by Ronolibert** [comment]
@sojeri We don't support the profile not being completed since we assume that when we get to the home screen it has at least the profile completed. In [this pr](https://github.com/transcom/mymove/pull/4711) is where it gets integrated to after the profile completion

File: src/pages/MyMove/Home/index.jsx [hunk 3]
```diff
@@ -11,109 +12,198 @@ import Step from 'components/Customer/Home/Step';
...
+        </ul>
+      );
+    }
+
+    if (!this.hasShipments) {
+      return (
+        <ul>
+          {this.renderHelperListItems([
+            'Preferred moving details',
+            'Destination address (your new place, your duty station ZIP, or somewhere else',
```
Comment (sojeri) [inline src/pages/MyMove/Home/index.jsx:88]: may I have a close paren please?

**Pull request merged**